Runtime/Scripts/VideoStream.cs

```diff
@@ -113,14 +113,15 @@ public IEnumerator Update()
                 }
                 var rgba = VideoBuffer.ToRGBA();
```

Comment:
You might consider using [*using var*](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/using) syntax here; this works since the root class of `RGBABBuffer` implements `IDisposable`.